### DISCUSSION TITLE
Heptane pool fire (not right hrr)

### DISCUSSION THREAD

**Discussion by HunterXHunter1997** [firemodels/fds#13865] [Q&A]
hello team,
I am using FDS 6.9.1 to test a heptane pool fire (1mx1m) and there is an opening at the top and the theoretical hrr is 2.4MW but the hrr for the pool raised very slowly, is there any recommended to change in my definitions?
please take a look at this photo on the link:
![image](https://github.com/user-attachments/assets/90399df6-7e9c-42c2-b62d-302e56329a87)

&HEAD CHID='heptane_pool_fire'/
&TIME T_END=100.0/
&DUMP DT_RESTART=300.0, DT_SL3D=0.25/

&MESH ID='Mesh05-01', IJK=40,40,50, XB=0.0,1.0,0.0,1.0,0.0,1.25/
&MESH ID='Mesh05-02', IJK=40,40,50, XB=0.0,1.0,0.0,1.0,1.25,2.5/
&MESH ID='Mesh05-03', IJK=40,40,50, XB=0.0,1.0,0.0,1.0,2.5,3.75/
&MESH ID='Mesh05-04', IJK=40,40,50, XB=0.0,1.0,0.0,1.0,3.75,5.0/
&MESH ID='Mesh05-05', IJK=40,40,50, XB=0.0,1.0,0.0,1.0,5.0,6.25/
&MESH ID='Mesh05-06', IJK=40,40,50, XB=0.0,1.0,0.0,1.0,6.25,7.5/
&MESH ID='Mesh05-07', IJK=40,40,50, XB=0.0,1.0,0.0,1.0,7.5,8.75/
&MESH ID='Mesh05-08', IJK=40,40,50, XB=0.0,1.0,0.0,1.0,8.75,10.0/
&MESH ID='Mesh05-09', IJK=40,40,50, XB=0.0,1.0,1.0,2.0,0.0,1.25/
&MESH ID='Mesh05-10', IJK=40,40,50, XB=0.0,1.0,1.0,2.0,1.25,2.5/
&MESH ID='Mesh05-11', IJK=40,40,50, XB=0.0,1.0,1.0,2.0,2.5,3.75/
&MESH ID='Mesh05-12', IJK=40,40,50, XB=0.0,1.0,1.0,2.0,3.75,5.0/
&MESH ID='Mesh05-13', IJK=40,40,50, XB=0.0,1.0,1.0,2.0,5.0,6.25/
&MESH ID='Mesh05-14', IJK=40,40,50, XB=0.0,1.0,1.0,2.0,6.25,7.5/
&MESH ID='Mesh05-15', IJK=40,40,50, XB=0.0,1.0,1.0,2.0,7.5,8.75/
&MESH ID='Mesh05-16', IJK=40,40,50, XB=0.0,1.0,1.0,2.0,8.75,10.0/
&MESH ID='Mesh05-17', IJK=40,40,50, XB=1.0,2.0,0.0,1.0,0.0,1.25/
&MESH ID='Mesh05-18', IJK=40,40,50, XB=1.0,2.0,0.0,1.0,1.25,2.5/
&MESH ID='Mesh05-19', IJK=40,40,50, XB=1.0,2.0,0.0,1.0,2.5,3.75/
&MESH ID='Mesh05-20', IJK=40,40,50, XB=1.0,2.0,0.0,1.0,3.75,5.0/
&MESH ID='Mesh05-21', IJK=40,40,50, XB=1.0,2.0,0.0,1.0,5.0,6.25/
&MESH ID='Mesh05-22', IJK=40,40,50, XB=1.0,2.0,0.0,1.0,6.25,7.5/
&MESH ID='Mesh05-23', IJK=40,40,50, XB=1.0,2.0,0.0,1.0,7.5,8.75/
&MESH ID='Mesh05-24', IJK=40,40,50, XB=1.0,2.0,0.0,1.0,8.75,10.0/
&MESH ID='Mesh05-25', IJK=40,40,50, XB=1.0,2.0,1.0,2.0,0.0,1.25/
&MESH ID='Mesh05-26', IJK=40,40,50, XB=1.0,2.0,1.0,2.0,1.25,2.5/
&MESH ID='Mesh05-27', IJK=40,40,50, XB=1.0,2.0,1.0,2.0,2.5,3.75/
&MESH ID='Mesh05-28', IJK=40,40,50, XB=1.0,2.0,1.0,2.0,3.75,5.0/
&MESH ID='Mesh05-29', IJK=40,40,50, XB=1.0,2.0,1.0,2.0,5.0,6.25/
&MESH ID='Mesh05-30', IJK=40,40,50, XB=1.0,2.0,1.0,2.0,6.25,7.5/
&MESH ID='Mesh05-31', IJK=40,40,50, XB=1.0,2.0,1.0,2.0,7.5,8.75/
&MESH ID='Mesh05-32', IJK=40,40,50, XB=1.0,2.0,1.0,2.0,8.75,10.0/

&REAC ID='Reaction1',
      FUEL='N-HEPTANE',
      AUTO_IGNITION_TEMPERATURE=0.0,
      CO_YIELD=0.05,
      SOOT_YIELD=0.04,
      HEAT_OF_COMBUSTION=4.4E+4,
      RADIATIVE_FRACTION=0.35/

&MATL ID='heptane',
      SPECIFIC_HEAT=2.24,
      CONDUCTIVITY=0.14,
      DENSITY=675.0,
      ABSORPTION_COEFFICIENT=187.5,
      EMISSIVITY=0.95,
      HEAT_OF_COMBUSTION=4.4E+4,
      HEAT_OF_REACTION=317.0,
      SPEC_ID(1,1)='N-HEPTANE',
      NU_SPEC(1,1)=1.0,
      BOILING_TEMPERATURE=98.35/

&SURF ID='fuel',
      RGB=146,202,166,
      MATL_ID(1,1)='heptane',
      MATL_MASS_FRACTION(1,1)=1.0,
      THICKNESS(1)=0.1/

&OBST ID='Obstruction', XB=0.5,1.5,0.5,1.5,-7.21645E-16,0.2, SURF_IDS='fuel','INERT','INERT'/ 

&VENT ID='Vent', SURF_ID='OPEN', XB=4.440892E-16,2.0,3.330669E-15,2.0,10.0,10.0/ 

&TAIL /

**Comment by drjfloyd** [comment]
You have a 1 m diameter hetpane pool inside of a 2 m x 2 m x 10 m shaft with closed sides. This is not a favorable geometry. Air is going to have a difficult time reaching the top of the pool. If burning is not able to occur at the pool top then you will have a lower heat feedback to the pool

Your heptane pool is liquid floating in the air.  You surf has no pan that the pool is stting in nor the ground that the fuel pan is sitting on. So rather than a insulative backside boundary condition, your backside condition is VOID.  This is going to increase heat losses of the back compared to the heat losses that would have existed in whatever reference source you are using.

&SURF ID='fuel',
RGB=146,202,166,
MATL_ID(1,1)='heptane',
MATL_MASS_FRACTION(1,1)=1.0,
THICKNESS(1)=0.1/

You are predicting rather than prescribing the heat release rate. Have you looked at the pool fire examples in the Validation Guide to see how well those predict the HRR? Any errors in predicting the heat feedback will directly translate to errors in the burning rate.

**Comment by HunterXHunter1997** [comment]
Thanks for your response @drjfloyd 
Okay, I rebuilt the model again with a floor and pan for the pool, but I had the same results. Also, as you mentioned, I made the surface "steel" insulated.
but I am wondering about something. when I used the same code in FDS 6.7.1 the burning rate was good, is there any significant change between the old and the new version that I have to take it into consideration when I model a pool fire?

here is my new model:

&HEAD CHID='new_test'/
&TIME T_END=100.0/
&DUMP COLUMN_DUMP_LIMIT=.TRUE., DT_RESTART=300.0, DT_SL3D=0.25/

&MESH ID='Mesh01', IJK=25,28,41, XB=3.5,6.0,3.75,6.5,-0.1,4.0/

&SPEC ID='N-HEPTANE'/

&REAC ID='Reaction1',
      FUEL='N-HEPTANE',
      AUTO_IGNITION_TEMPERATURE=0.0,
      CO_YIELD=6.0E-3,
      SOOT_YIELD=0.015,
      HEAT_OF_COMBUSTION=4.4E+4,
      RADIATIVE_FRACTION=0.35/

&MATL ID='STEEL',
      FYI='Drysdale, Intro to Fire Dynamics - ATF NIST Multi-Floor Validation',
      SPECIFIC_HEAT=0.46,
      CONDUCTIVITY=45.8,
      DENSITY=7850.0,
      EMISSIVITY=0.95/
&MATL ID='heptane',
      SPECIFIC_HEAT=2.24,
      CONDUCTIVITY=0.14,
      DENSITY=675.0,
      ABSORPTION_COEFFICIENT=187.5,
      EMISSIVITY=1.0,
      HEAT_OF_REACTION=318.0,
      SPEC_ID(1,1)='N-HEPTANE',
      NU_SPEC(1,1)=1.0,
      BOILING_TEMPERATURE=98.35/

&SURF ID='steel',
      COLOR='BLACK',
      BACKING='INSULATED',
      MATL_ID(1,1)='STEEL',
      MATL_MASS_FRACTION(1,1)=1.0,
      THICKNESS(1)=1.73E-3/
&SURF ID='heptane',
      COLOR='RED',
      MATL_ID(1,1)='heptane',
      MATL_MASS_FRACTION(1,1)=1.0,
      THICKNESS(1)=0.12/

&OBST ID='Obstruction', XB=3.5,4.3,4.633929,5.714286,-0.1,-1.387779E-17, SURF_ID='steel'/ 
&OBST ID='Obstruction', XB=3.5,6.0,3.75,4.633929,-0.1,-1.387779E-17, SURF_ID='steel'/ 
&OBST ID='Obstruction', XB=3.5,6.0,5.714286,6.5,-0.1,-1.387779E-17, SURF_ID='steel'/ 
&OBST ID='Obstruction', XB=4.3,5.3,4.633929,5.714286,-0.1,0.2, SURF_IDS='heptane','steel','steel'/ 
&OBST ID='Obstruction', XB=5.3,6.0,4.633929,5.714286,-0.1,-1.387779E-17, SURF_ID='steel'/ 
&OBST ID='Obstruction', XB=4.3,5.3,4.633929,4.633929,0.2,0.3, SURF_ID='steel'/ 
&OBST ID='Obstruction', XB=4.3,5.3,5.714286,5.714286,0.2,0.3, SURF_ID='steel'/ 
&OBST ID='Obstruction', XB=4.3,4.3,4.633929,5.714286,0.2,0.3, SURF_ID='steel'/ 
&OBST ID='Obstruction', XB=5.3,5.3,4.633929,5.714286,0.2,0.3, SURF_ID='steel'/ 

&VENT ID='Mesh Vent: Mesh01 [XMAX]', SURF_ID='OPEN', XB=6.0,6.0,3.75,6.5,-0.1,4.0/ 
&VENT ID='Mesh Vent: Mesh01 [XMIN]', SURF_ID='OPEN', XB=3.5,3.5,3.75,6.5,-0.1,4.0/ 
&VENT ID='Mesh Vent: Mesh01 [YMAX]', SURF_ID='OPEN', XB=3.5,6.0,6.5,6.5,-0.1,4.0/ 
&VENT ID='Mesh Vent: Mesh01 [YMIN]', SURF_ID='OPEN', XB=3.5,6.0,3.75,3.75,-0.1,4.0/ 
&VENT ID='Mesh Vent: Mesh01 [ZMAX]', SURF_ID='OPEN', XB=3.5,6.0,3.75,6.5,4.0,4.0/ 


&TAIL /

**Reply by mcgratta** [reply]
You can look at the [Pool_Fires validation cases](https://github.com/firemodels/fds/tree/master/Validation/Pool_Fires/FDS_Input_Files) that are described in the FDS Validation Guide.
